### PR TITLE
Manifested Ghosts are now actually immune to stuns

### PR DESCRIPTION
**Pull request by ShiftyRail**
[dnm] [balance] [bugfix]

Don't merge this, because I'm not actually sure if this is a bugfix or not.
The wiki page on manifested ghost tattoo (Pale Body) says that you'll no longer feel pain AND be immune to stun.
The in-game description just says that you're immune to pain.
So this would be a bugfix, if this was the intended change.

If it's not a bugfix, (ie, the wiki is wrong) then I would close this PR because I think that pale ghosts would be very, very strong with that advantage.

:cl:
- bugfix: Manifested Ghosts are now actually stun immune.

### CODE REVIEW

**Comment by I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE** [comment]
https://github.com/vgstation-coders/vgstation13/blob/Bleeding-Edge/__DEFINES/setup.dm#L572

**Comment by Irradiation** [comment]
Ye who enter, abandon all hope...

**Comment by Durgen** [comment]
Are manifested ghosts still spamable?

**Comment by iFlashYou** [comment]
@Durgen How about you play the game first?

**Comment by Durgen** [comment]
oh no, it appears I have been yeeted

**Comment by ShiftyRail** [comment]
@Durgen They're a new and upgraded version of manifested ghosts which are available near the end of Cult III, when cultists have almost won.

**Comment by ShiftyRail** [comment]
This isn't a bug, but I need to use the existing flag

**Comment by ShiftyRail** [comment]
:cl: added

File: code/datums/gamemode/factions/bloodcult/bloodcult_tattoos.dm [hunk 1]
```diff
@@ -111,6 +111,10 @@ var/list/blood_communion = list()
 	H.b_eyes = 0
 	H.revive(0)
 	H.status_flags &= ~GODMODE
```
Comment (MadmanMartian) [inline code/datums/gamemode/factions/bloodcult/bloodcult_tattoos.dm:113]: Why does this specifically remove godmode?

**Review by MadmanMartian — APPROVED** [review]

File: code/datums/gamemode/factions/bloodcult/bloodcult_tattoos.dm (hunk 1 above)
Comment (ShiftyRail) [reply code/datums/gamemode/factions/bloodcult/bloodcult_tattoos.dm:113]: Manifested ghosts have Godmode by default

**Pull request merged**